R/Attributes.R

```diff
@@ -67,7 +67,7 @@ sourceCpp <- function(file = "",
     } else {
         if (windowsDebugDLL) {
             if (verbose) {
-                message("The 'windowsDebugDLL' toggle is ignored on "
+                message("The 'windowsDebugDLL' toggle is ignored on ",
```

Comment:
Good catch on this one.

Reply:
Well it was a last-second edit to comply with the @jjallaire style of less than 80 columns. I find 100 or so easier and this is what I got for it :)